sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
+	"Resources": {
+		"ServiceRoot": {
+			"PropertyRequirements": {
+				"Chassis": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ChassisCollection"]
+				},
+				"Managers": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ManagerCollection"]
+				},
+				"SessionService": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["SessionService"]
+				},
+				"Id": {
+					"ReadRequirement": "Mandatory"
+				},
+				"Name": {
+					"ReadRequirement": "Recommended"
+				},
+				"UUID": {
+					"ReadRequirement": "Recommended"
+				},
+				"Product": {
+					"ReadRequirement": "Recommended"
+				},
+				"ProtocolFeaturesSupported": {
+					"PropertyRequirements": {
+						"ExcerptQuery" : {
+							"ReadRequirement" : "Recommended"
+						},
+						"ExpandQuery" : {
+							"ReadRequirement" : "Recommended"
+						},
+						"FilterQuery" : {
+							"ReadRequirement" : "Recommended"
+						},
+						"OnlyMemberQuery" : {
+							"ReadRequirement" : "Recommended"
+						},
+						"SelectQuery" : {
+							"ReadRequirement" : "Recommended"
+						}
+					}
+				},
+				"Systems": {
```

Comment:
As with `ProtocolFeaturesSupported` above, I believe this property's `ReadRequirement` is `Recommended` and therefore must be explicitly provided.